.github/workflows/cargo-fmt.yml

```diff
@@ -32,3 +32,5 @@ jobs:
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
       - name: Check formatting
         run: cargo fmt --check
+      - name: Clippy
+        run: cargo clippy -- -D warnings
```

Comment:
We should probably rename the workflow or put this after a `cargo check`/`cargo build` as they have a lot of overlap.

Reply:
This needs to go after a `cargo build` more than I thought, since it needs to build the code, so it needs things like `nasm`, which aren't necessary for `cargo fmt` and thus not installed in this workflow.